pkg/stdlib/driver.go

```diff
@@ -44,44 +45,60 @@ func OpenDB(cliOpts *client.Options) *sql.DB {
 	return sql.OpenDB(c)
 }
 
+/*func Open(dns string) *sql.DB {
```

Comment:
can be removed?

Reply:
I will restore it
db, err := sql.Open("immudb", "immudb://immudb:immudb@127.0.0.1:3322/defaultdb?sslmode=disable")
or 
db := Open("immudb://immudb:immudb@127.0.0.1:3322/defaultdb?sslmode=disable")

are equivalent